geopandas/base.py

```diff
@@ -2721,14 +2721,25 @@ def simplify(self, *args, **kwargs):
         """Returns a ``GeoSeries`` containing a simplified representation of
         each geometry.
 
+        The algorithm (Douglas-Peucker) recursively splits the original line
+        into smaller parts and connects these parts’ endpoints
+        by a straight line. Then, it removes all points whose distance
+        to the straight line is smaller than `tolerance`. It does not
+        move any points and it always preserves endpoints of
+        the original line or polygon.
         See http://shapely.readthedocs.io/en/latest/manual.html#object.simplify
         for details
 
         Parameters
         ----------
         tolerance : float
-            All points in a simplified geometry will be no more than
-            `tolerance` distance from the original.
+            All parts of a simplified geometry will be no more than
+            `tolerance` distance from the original. It has the same units
+            as the coordinate reference system of the GeoSeries.
+            For example, using `tolerance=100` in the Mercator projection means
```

Comment:
```suggestion
            For example, using `tolerance=100` in the projected CRS with meters as units
```

Something like this? Instead of Mercator? Then we can eliminate the note at the end.